mawabot/cogs/general.py

```diff
@@ -68,6 +68,13 @@ def __init__(self, bot):
             rolls = ' + '.join(rolls)
             await ctx.send(content=f'🎲 {rolls} = {total}')
 
+    @commands.command()
```

Comment:
Add `@comamnds.guild_only()` to make the command only work in guilds and not DM as they will cause it to break